src/App.tsx

```diff
@@ -1,12 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { GoodsList } from './components/GoodsList';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-const App: React.FC = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
+
+  return (
+    <div className="App box">
+      <h1 className="title">Dynamic list of Goods</h1>
+
+      <button
+        className="button"
+        type="button"
+        onClick={async () => setGoods(await getAll())}
```

Comment:
create separate function for each button

Reply:
agree, let's create a separate functions for onClick

Reply:
I agree, thank you very much for checking the assignment and feedback))